src/client/components/features/Accompagnement/Rechercher/Résultat/RésultatRechercherAccompagnement.tsx

```diff
@@ -1,48 +1,119 @@
+import classNames from 'classnames';
 import React, { useCallback, useState } from 'react';
 
 import {
 	ModalDemandeDeContactAccompagnement,
 } from '~/client/components/features/Accompagnement/DemandeDeContact/ModalDemandeDeContactAccompagnement';
 import {
-	RésultatRechercherAccompagnementDesktop,
-} from '~/client/components/features/Accompagnement/Rechercher/Résultat/RésultatRechercherAccompagnementDesktop';
+	HorairesResultatRechercherAccompagnement,
+} from '~/client/components/features/Accompagnement/Rechercher/Résultat/Horaires/HorairesResultatRechercherAccompagnement';
 import {
-	RésultatRechercherAccompagnementMobile,
-} from '~/client/components/features/Accompagnement/Rechercher/Résultat/RésultatRechercherAccompagnementMobile';
-import useBreakpoint from '~/client/hooks/useBreakpoint';
+	RésultatRechercherAccompagnementTagsList,
+} from '~/client/components/features/Accompagnement/Rechercher/Résultat/RésultatRechercherAccompagnementTagsList';
+import {
+	useAccompagnementLogo,
+} from '~/client/components/features/Accompagnement/Rechercher/Résultat/useAccompagnementLogo';
+import { ButtonComponent } from '~/client/components/ui/Button/ButtonComponent';
+import { Card } from '~/client/components/ui/Card/Card';
+import { Link } from '~/client/components/ui/Link/Link';
 import {
 	EtablissementAccompagnement,
 	TypeÉtablissement,
 } from '~/server/etablissement-accompagnement/domain/etablissementAccompagnement';
 
+import styles from './RésultatRechercherAccompagnement.module.scss';
+
 export interface RésultatRechercherAccompagnementProps {
 	établissement: EtablissementAccompagnement
 }
 
 export function RésultatRechercherAccompagnement({ établissement }: RésultatRechercherAccompagnementProps) {
-	const { isLargeScreen } = useBreakpoint();
 
 	const isMissionLocale = établissement.type === TypeÉtablissement.MISSION_LOCALE;
 	const [isPopInOpen, setIsPopInOpen] = useState(false);
+	const logoÉtablissement = useAccompagnementLogo(établissement.type);
 
 	const openContactÉtablissementModal = useCallback(() => {
 		setIsPopInOpen(true);
 	}, []);
 
+
+	function ContactButton() {
+		const label = 'Je souhaite être contacté(e)';
+		return <>
+			<ButtonComponent
+				className={classNames(styles.contactFormulaireÉtablissement, styles.contactFormulaireÉtablissementDesktopOnly)}
+				label={label}
+				appearance={'quaternary'}
+				onClick={openContactÉtablissementModal}/>
+
+			<ButtonComponent
+				className={classNames(styles.contactFormulaireÉtablissement, styles.contactFormulaireÉtablissementMobileOnly)}
+				label={label}
+				appearance={'primary'}
+				onClick={openContactÉtablissementModal}/>
+		</>;
+	}
+
+	function MailLink(props: { établissement: EtablissementAccompagnement }) {
+		const label = 'Contacter l‘agence';
+		const mailTo = `mailto:${props.établissement.email}`;
+		const title = 'Contacter l‘agence - adresse mail';
```

Comment:
```suggestion
		const title = `${label} - adresse mail`;
```

(Je pousse le bouchon un peu loin mais j'aurais peur qu'on modifie le label en oubliant le title :D )